kimchi/src/circuits/polynomials/range_check/witness.rs

```diff
@@ -192,11 +203,20 @@ pub fn create_multi_witness<F: PrimeField>(v0: F, v1: F, v2: F) -> [Vec<F>; COLU
 }
 
 /// Create a single range check witness
-/// Input: three values: v0, v1 and v2
+/// Input: 88-bit value v0
 pub fn create_witness<F: PrimeField>(v0: F) -> [Vec<F>; COLUMNS] {
     let mut witness: [Vec<F>; COLUMNS] = array_init(|_| vec![F::zero(); 4]);
 
     init_range_check_row(&mut witness, 0, v0);
 
     witness
 }
+
+/// Extend an existing witness with a multi-range-check gate for foreign field
+/// elements fe
+pub fn extend_witness<F: PrimeField>(witness: &mut [Vec<F>; COLUMNS], fe: ForeignElement<F, 3>) {
+    let limbs_witness = create_multi_witness(*fe.lo(), *fe.mi(), *fe.hi());
+    for col in 0..COLUMNS {
+        witness[col].extend(limbs_witness[col].iter())
+    }
+}
```

Comment:
not used anywhere no?

Reply:
Oh well, you're right it is not used in this PR, but it will be used for FFMul and FFAdd, I can remove it from this PR.